src/StartMenu.ts

```diff
@@ -335,40 +343,27 @@ export class AEntryPointList implements IEntryPointList {
     const that = this;
 
     const predefinedNamedSets = data.filter((d) => d.type !== ENamedSetType.NAMEDSET);
-    const customNamedSets = data.filter((d) => d.type === ENamedSetType.NAMEDSET);
+    const me = currentUserNameOrAnonymous();
+    const customNamedSets = data.filter((d) => d.type === ENamedSetType.NAMEDSET && d.creator === me);
+    const otherNamedSets = data.filter((d) => d.type === ENamedSetType.NAMEDSET && d.creator !== me);
```

Comment:
What if I create a named set and set it to public: should it be shown in "My Named Sets", in "Public Named Sets" or in both?